src/miniz/miniz.h

```diff
@@ -1,3 +1,9 @@
+// Copyright 2001-2017,
+// Daniel Kroening (Computer Science Department, University of Oxford
+//   and Diffblue Ltd),
+// Edmund Clarke (Computer Science Department, Carnegie Mellon University),
+// DiffBlue Ltd.
+
```

Comment:
This change is not appropriate (and the same is true of miniz.cpp, as well as big-int/*).